optuna/pruners/hyperband.py

```diff
@@ -224,3 +228,48 @@ def __getattribute__(self, attr_name):  # type: ignore
                     return object.__getattribute__(self, attr_name)
 
         return _BracketStudy(study, bracket_index)
+
+
+class _HyperbandSampler(BaseSampler):
```

Comment:
Please add the [`reseed_rng`](https://github.com/optuna/optuna/blob/master/optuna/samplers/base.py#L124-L134) method so that wrapped samplers correctly work with multi-processing.

```python
def reseed_rng(self) -> None:
    self._sampler.reseed_rng()
```

Reply:
Thank you for your comment. I will add it.